libs/portals/my-pages/core/src/components/Charts/sharedChartComponents.tsx

```
@@ -0,0 +1,199 @@
+  }
+
+  return (
+    <Box
+      as="ul"
+      marginTop={3}
+      display="flex"
+      width="full"
+      justifyContent={'flexEnd'}
+    >
+      {[...payload.values()].map((item) => (
+        <Box display="flex" as="li">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)